common.php

```diff
@@ -258,7 +258,7 @@ function display_footer() {
 			</div>
 			<div class="row">
 				<div class="col text-center">
-					<a href="https://github.com/ku-coursecorrect/coursecorrect">CourseCorrect</a> Copyright &copy; 2022: Drake Prebyl, James Kraijcek, Rafael Alaras, Reece Mathews, Tiger Ruan
+					<a href="https://github.com/ku-coursecorrect/coursecorrect">CourseCorrect</a> Copyright &copy; 2022: Drake Prebyl, James Krajicek, Rafael Alaras, Reece Mathews, Tiger Ruan
```

Comment:
Whoops

Reply:
Haha. Not a problem, you got closer than 90% of my other peers!